File: zariodecomp/app/src/main/java/com/niyaz/zario/ui/about/AboutFragment.java
package com.niyaz.zario.ui.about;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ScrollView;
import b2.ViewOnClickListenerC0262a;
import com.niyaz.zario.databinding.FragmentAboutBinding;
import g0.AbstractComponentCallbacksC0567t;
import kotlin.jvm.internal.i;

/* loaded from: C:\Users\niyaz\Downloads\universal\smali\com\niyaz\zario\ui\about\AboutFragment.smali */
public final class AboutFragment extends AbstractComponentCallbacksC0567t {

    /* renamed from: a0, reason: collision with root package name */
    public FragmentAboutBinding f6665a0;

    @Override // g0.AbstractComponentCallbacksC0567t
    public final View E(LayoutInflater inflater, ViewGroup viewGroup) {
        i.e(inflater, "inflater");
        FragmentAboutBinding fragmentAboutBindingInflate = FragmentAboutBinding.inflate(inflater, viewGroup, false);
        this.f6665a0 = fragmentAboutBindingInflate;
        i.b(fragmentAboutBindingInflate);
        ScrollView root = fragmentAboutBindingInflate.getRoot();
        i.d(root, "getRoot(...)");
        return root;
    }

    @Override // g0.AbstractComponentCallbacksC0567t
    public final void G() {
        this.f7510H = true;
        this.f6665a0 = null;
    }

    @Override // g0.AbstractComponentCallbacksC0567t
    public final void O(View view) {
        i.e(view, "view");
        FragmentAboutBinding fragmentAboutBinding = this.f6665a0;
        i.b(fragmentAboutBinding);
        fragmentAboutBinding.btnContact.setOnClickListener(new ViewOnClickListenerC0262a(this, 4));
    }
}

File: app/src/main/java/com/niyaz/zario/ui/about/AboutFragment.kt
package com.niyaz.zario.ui.about

import android.content.ActivityNotFoundException
import android.content.Intent
import android.net.Uri
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import androidx.lifecycle.lifecycleScope
import com.google.android.material.snackbar.Snackbar
import com.niyaz.zario.Constants
import com.niyaz.zario.R
import com.niyaz.zario.data.Condition
import com.niyaz.zario.databinding.FragmentAboutBinding
import com.niyaz.zario.repository.UserSessionRepository
import dagger.hilt.android.AndroidEntryPoint
import javax.inject.Inject
import kotlinx.coroutines.flow.collectLatest
import kotlinx.coroutines.launch

@AndroidEntryPoint
class AboutFragment : Fragment() {

    private var _binding: FragmentAboutBinding? = null
    private val binding get() = _binding!!

    @Inject lateinit var sessionRepository: UserSessionRepository

    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View {
        _binding = FragmentAboutBinding.inflate(inflater, container, false)
        return binding.root
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        binding.btnContact.setOnClickListener { sendContactEmail() }
        observeSession()
    }

    private fun observeSession() {
        viewLifecycleOwner.lifecycleScope.launch {
            sessionRepository.session.collectLatest { session ->
                val condition = session.user?.condition
                val showPoints = condition != null && condition != Condition.BENCHMARK
                binding.tvEvaluation.text =
                    if (showPoints) getString(R.string.about_evaluation_process)
                    else getString(R.string.about_evaluation_process_benchmark)
            }
        }
    }

    private fun sendContactEmail() {
        val emailAddress = Constants.CONTACT_EMAIL_SUPPORT
        val emailIntent = Intent(Intent.ACTION_SENDTO).apply {
            data = Uri.parse("mailto:$emailAddress")
            putExtra(Intent.EXTRA_EMAIL, arrayOf(emailAddress))
            putExtra(Intent.EXTRA_SUBJECT, getString(R.string.about_title))
        }

        try {
            startActivity(Intent.createChooser(emailIntent, getString(R.string.about_contact_chooser_title)))
        } catch (ex: ActivityNotFoundException) {
            view?.let { root ->
                Snackbar.make(root, R.string.about_contact_error, Snackbar.LENGTH_SHORT).show()
            }
        }
    }

    override fun onDestroyView() {
        super.onDestroyView()
        _binding = null
    }
}
